vcr/credential/builder.go

```diff
@@ -42,7 +42,8 @@ type defaultBuilder struct {
 	vcType string
 }
 
-var nowFunc = time.Now
+// NowFunc is used to store a function that returns the current time. This can be changed when you want to mock the current time.
+var NowFunc = time.Now
```

Comment:
I'm guessing this is because some test from another package needs it. Still not very nice to expose something only changed for test...

Reply:
Hmm I thought I reverted this change, was not meant to be changed after all.